specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -2668,7 +2720,24 @@
         "typeProperties": {
           "description": "Properties specific to this dataset type.",
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/GenericDatasetTypeProperties"
+          "$ref": "#/definitions/NetezzaTableDatasetTypeProperties"
```

Comment:
is it a break change?

Reply:
there are a bunch of similar changes made before. One reference: https://github.com/Azure/azure-rest-api-specs/commit/74dfcd34b63a19fcae30518bafdddd8f7c1c0dbd

Reply:
Mainly a change of name. Thanks